manual/development-process/test-accessibility.md

```diff
@@ -0,0 +1,99 @@
+# Test the accessibility of a feature
+
+Transfer Digital Records needs to meet the [accessibility requirements for
+public sector bodies][a11y-reqs], which means that every page and feature needs
+to pass at least the AA level of the [Web Content Accessibility Guidelines
+(WCAG)][wcag].
+
+We think about accessibility when designing new features, and we periodically
+check the accessibility of the whole site. Developers also need to test for
+accessibility when building new features. This checklist is a subset of the full
+WCAG AA standard. It includes checks which are particularly relevant to TDR,
+and which can be checked fairly quickly by developers. We should keep this
+checklist up-to-date as we learn more about testing TDR or if we add completely
+new types of features, such as multimedia content.
+
+[a11y-reqs]: https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps
+[wcag]: https://www.w3.org/WAI/standards-guidelines/wcag/
+
+## What to test
+
+Some of these tests are subjective. For example, what is the most obvious and
+logical tab order? If you are not sure about anything, talk to the rest of the
+team to get other people's opinions, and to check whether it's something that
+we should test with users.
+
+### Manual tests
+
+* Zoom into 400% and check the page is still readable and functional
+* Disable images and styles (search for how to do this in your specific browser)
+  and check that the page is still readable and functional
+* Check all functionality works using only the keyboard, and that you don't get
+  stuck when tabbing through elements
+* Check that it's obvious which element has focus when tabbing to it with the
+  keyboard
+* Check that keyboard tab order is obvious
+* Check that semantic markup is used wherever possible (e.g. `<ul>` rather than
+  `<div>` for a list, or `<h1>` rather than a styled `<span>` for a header)
+
+### Screen reader tests
+
+Navigate through the page using a screen reader. Choose one which works on your
+operating system:
+
+* [VoiceOver] for MacOS
+* [NVDA] for Windows
+* [Orca] for Linux
+
+What to test with with the screen reader:
+
+* Check that the screen reader reads every element and header
+* Check that every link makes sense in the screen reader
+* Check that [ARIA] roles are being used where appropriate
+* Check that and that you can fill in every editable field
```

Comment:
I think this might be a typo? `Check that and that you can`

Reply:
Fixed, thanks!